addons/account/models/account_move_line.py

```diff
@@ -415,6 +415,10 @@ class AccountMoveLine(models.Model):
              "associated partner",
     )
     is_refund = fields.Boolean(compute='_compute_is_refund')
+    # === Check Info === #
+    check_number = fields.Char(
+        store=False,
+        compute='_get_check_number')
```

Comment:
i would put the parenthesis on the next line and put a comma at the end of the compute. It will allow the next person that will modify the field to not break the github history on the line :smile: 

Also compute usually start with _compute so that we know directly that it's a compute function